src/app/screens/Options/TestConnection/index.tsx

```diff
@@ -11,6 +11,7 @@ import api from "~/common/lib/api";
 import msg from "~/common/lib/msg";
 import type { AccountInfo } from "~/types";
 
+// alais should be replaced with connector
```

Comment:
is this comment needed?

Reply:
considering migration of existing account. left this comment here. but no more the case as we use two separate functions. removed